### DISCUSSION TITLE
v2.225

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#547] [Announcements]
Important changes
1. Search bar added in YouTube RVX settings. 
2. Client spoof added as a fix for playback issue
3. Force snackbar theme: color, corner radius, stroke color 
4. Ability to remove languages (thereby reduce the app size) and add your own strings.xml file in `Translations` patch

When I have time I merge changes from this repo to inotia's, so one day our repos are synced. 

As for now, you can check [unique features](https://github.com/anddea/revanced-patches/discussions/446) for this repo.

**Comment by Jaydan101** [comment]
Anddea, thank you so much. Please merge your settings icons with inotia00. Those settings icon and search feature are so perfect! Thank you for all that you do 🙏🏿

**Comment by mryx007** [comment]
but where is the patch ‘hide channel avatar’? In addition, the search bar is no longer wide. There is now the Youtube logo on the left side

**Reply by anddea** [reply]
The first one is called `Hide subscriptions channel section`. The second one is made as a feature request to show logo alongside with wide search bar, there's a second option for wide search bar with header, just disable it. You can use search in settings and you will find them immediately.

**Comment by mryx007** [comment]
Thanks, I was able to find everything. The search function is great and I love the icons.

**Comment by odkate** [comment]
@anddea Why did you enter strings in the missing strings?
Some of them are in the translate, but with the tag `formatted="false" `
The rest were Translated on the crowdin yesterday

**Reply by anddea** [reply]
Automated script is somewhat incompatible after refactoring. I updated it a little.
If inotia did not merge translations yet, they won't be here as well, as I never work with Crowdin, I only work with Github.

**Reply by odkate** [reply]
> work with Github

From your answer, it's not clear whether you need to create Pull requests or not?

**Reply by anddea** [reply]
No, it will be merged later, when inotia pushes your updates. Although, I contributed everything (I think), so there's no point to come back here to this repo anymore.

**Comment by manojkumarcrazy** [comment]
> Important changes
> 1. Search bar added in YouTube RVX settings. 
> 2. Client spoof added as a fix for playback issue
> 3. Force snackbar theme: color, corner radius, stroke color 
> 4. Ability to remove languages (thereby reduce the app size) and add your own strings.xml file in `Translations` patch
> 
> When I have time I merge changes from this repo to inotia's, so one day our repos are synced. 
> 
> As for now, you can check [unique features](https://github.com/anddea/revanced-patches/discussions/446) for this repo. 

YouTube font different color option patch will for next update

**Reply by anddea** [reply]
Not in my plans yet
